.env

```diff
@@ -14,6 +14,5 @@ REACT_APP_USE_DOPPLER_LEGACY_FORGOTPASSWORD=false
 NODE_ICU_DATA=node_modules/full-icu
 REACT_APP_ROUTER=browser
 REACT_APP_DOPPLER_UI_LIBRARY_VERSION = v3.89.1
-REACT_APP_DATA_HUB_URL= https://hub.fromdoppler.com
```

Comment:
Please do not remove this since we use it for datahub. (to track)

Reply:
Done!